### ISSUE TITLE
Discrete  GPU acceleration

### ISSUE THREAD

**Issue by naixx** [GreatAttractor/imppg#26]
Hey, I want to point an option to enable discrete GPU for imppg in your pc settings, in my case in Nvidia control panel. 

After I enabled 3080 instead of integrated graphics, processing became even faster.

Probably that should be mentioned in readme or be enabled from within the app

**Comment by GreatAttractor** [comment]
Good point. Could you post a screenshot of that NVidia tool's settings page?

My own laptop has an iGPU, but I'll check that behavior on the work laptop which has a Quadro.

**Comment by naixx** [comment]
![image](https://github.com/user-attachments/assets/1d5cd93f-2686-464f-8116-dc8218d3c49e)
Yeah, impgg install in deepskystacker
